powerauth-admin/src/main/java/io/getlime/security/app/admin/configuration/ApplicationConfiguration.java

```diff
@@ -55,6 +55,12 @@ public class ApplicationConfiguration {
     @Value("${powerauth.admin.service.applicationEnvironment}")
     private String applicationEnvironment;
 
+    /**
+     * Specifies the proportion of requests that are sampled for tracing.
+     */
+    @Value("${management.tracing.sampling.probability}")
+    private double tracingSamplingProbability;
```

Comment:
Since we do not use this value ourselves, declaring it in the configuration class is redundant.

Reply:
@banterCZ
I thought, the goal was to add variable to override default value, no? 

> yes we can have this turned on by default, but provide a way how to configure it. ie. 1 is default, but there has to be a variable to override it.